unimport/console.py

```diff
@@ -1,11 +1,63 @@
+import argparse
 import os
-import sys
+import pathlib
 import tokenize
 
+from unimport.config import Config
 from unimport.files import get_files
 from unimport.unused import get_unused
 
 
+class Cli(object):
+    def parse_args(self):
```

Comment:
Just for my own curiosity why you are re-constructing `ArgumentParser` in every time you call `parse_args`, shouldn't `ArgumentParser` be part of `Cli` class? In practice it wouldn't matter but someone who is reading code, I'd prefer to keep it in class body.

Reply:
@isidentical can you adapt this part after my PR is merged? I'm focusing on your other feedbacks.

Reply:
@gkmngrgn sure!